package.json

```diff
@@ -115,6 +109,7 @@
     "lerna": "^3.21.0",
     "lerna-changelog": "^1.0.1",
     "lint-staged": "^7.1.0",
+    "lodash.memoize": "^4.1.2",
```

Comment:
For some reason, removing cypress/code-coverage broke storybook, and npm wasn't installing this module correctly

Reply:
I feel like we'll need to replace storybook sometime soon